lib/phoenix_live_component.ex

```diff
@@ -434,14 +434,14 @@ defmodule Phoenix.LiveComponent do
 
   alias Phoenix.LiveView.Socket
 
-  defmacro __using__(_) do
+  defmacro __using__(opts \\ []) do
     quote do
       import Phoenix.LiveView
       @behaviour Phoenix.LiveComponent
       @before_compile Phoenix.LiveView.Renderer
 
       # Phoenix.Component must come last so its @before_compile runs last
-      use Phoenix.Component
+      use Phoenix.Component, opts
```

Comment:
you need to unquote the opts here, but we should only pass those specific to live_component `Keyword.take(unquote(opts), [:global_prefixes])`